ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -631,6 +634,10 @@ QtObject {
       function onCommunityAccessRequested(communityId) {
           root.communityAccessRequested(communityId)
       }
+
+      function onCommunityAdded(communityId) {
```

Comment:
It's wrapping the backend communitiesModuleInst in Connections and sending a RootStore signal. This is just to avoid qml code going directly to the nim backend.

Reply:
You could just emit the qml signal from the `Connections`?

Reply:
Hmm, this is what I'm doing. This function is part of the Connections object.

Please help me with a code snippet if I've missed the point 🍻

Reply:
Ah sorry, _I_ was missing the context here literally (didn't see this function is inside the `Connections`) :)